ln-simln-jamming/src/revenue_interceptor.rs

```diff
@@ -163,7 +163,14 @@ impl RevenueInterceptor {
     /// Replays projected forwards for the simulated network in a time of peace (ie, without the attacker). These events
     /// are replayed "live" so that they can be compared to the current simulation's revenue.
     pub async fn process_peacetime_fwds(&self) -> Result<(), BoxError> {
-        let mut last_event_ts = None;
+        let mut last_event_ts = self
+            .peacetime_revenue
+            .lock()
+            .await
+            .revenue_events
+            .pop()
```

Comment:
nit: use `peek` here instead? we're always going to drop our first event otherwise